### DISCUSSION TITLE
Support for .json

### DISCUSSION THREAD

**Discussion by pmaster** [evolutionsoftswiss/pokernow-handhistory-converter#21] [Q&A]
Is there currently support or any plans for support for Pokernow replayer .json files?

Currently, pokernow limits log file downloads to only the 20k most recent lines of the log, which expire after some days or weeks. These log files are in .csv format.

The replayer files don't expire, aren't limited to a subset of the hands, but are in a .json format.

Thank you for providing this.

**Comment by evolutionsoftswiss** [comment]
JSON files are not yet supported and I can't give a plan yet to support this.

**Comment by Deckhole** [comment]
PokerNow also supports OHH, which is much easier to parse.  Maybe you could consider this as an option?

**Reply by evolutionsoftswiss** [reply]
If pokernow newly provides OHH format, couldn't that get imported directly without conversion in the tracker tools ?

**Reply by Deckhole** [reply]
Yes, but their OHH export isn’t perfect and doesn’t support tournament yet.
